src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -51,72 +46,38 @@ const NewAppZeroState = ({
         return () => {
             mounted.current = false;
         };
-    }, [axios, customFetchResults, hasSystems]);
-
-    return (
-        hasSystems ? children :
-            <IntlProvider>
-                {app.toLowerCase() === 'dashboard'
-                    ? <ZeroState />
-                    :  <Fragment>
-                        <ZeroStateBanner
-                            appName={app}
-                            customInstructions={customInstructions}
-                            customButton={customButton}
-                            customText={customText}
-                            customTitle={customTitle}
-                            appId={appId}
-                        />
-                        {customSection && customSection}
-                        <AppSection appName={app}/>
-                        <ZeroStateFooter appName={app} />
-                    </Fragment>
-                }
-            </IntlProvider>
-    );
-};
+    }, [axios, children, customFetchResults, hasSystems]);
 
-//We will slowly migrate away from this
-const OldAppZeroState = ({
-    app,
-    customInstructions,
-    customButton,
-    customText,
-    customTitle,
-    appId
-}) => {
+    //If there are children, act as a wrapper, otherwise a component
     return (
         <IntlProvider>
-            <React.Fragment>
-                <ZeroStateBanner
-                    appName={app}
-                    customInstructions={customInstructions}
-                    customButton={customButton}
-                    customText={customText}
-                    customTitle={customTitle}
-                    appId={appId}
-                />
-                <AppSection appName={app}/>
-                <ZeroStateFooter appName={app} />
-            </React.Fragment>
+            {(children && hasSystems) ? (
+
+                children
+            ) : app.toLowerCase() === 'dashboard' ? (
```

Comment:
Do you think there is a way we can factor this out and make the "dashboard" app behave and use the "AppZeroState" like other apps?

If it is drastically different than another app zero state we should maybe just call the `ZeroState` component directly in Dashboard.

Reply:
Yeah dashboards zero state is completely different than the rest. I ll set that up to reduce conditionals here